thehive/info.json

```diff
@@ -0,0 +1,2241 @@
+          "description": "Specify the ID of the case in which you want to create task in TheHive Platform",
+          "tooltip": "Specify the ID of the case in which you want to create task in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": true
+        },
+        {
+          "title": "Title",
+          "name": "title",
+          "type": "text",
+          "description": "Specify the title of the task you want to create in TheHive Platform",
+          "tooltip": "Specify the title of the task you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": true
+        },
+        {
+          "title": "Description",
+          "name": "description",
+          "type": "text",
+          "description": "Specify the description of the task you want to create in TheHive Platform",
+          "tooltip": "Specify the description of the task you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Status",
+          "name": "status",
+          "type": "text",
+          "description": "Specify the status of the task you want to create in TheHive Platform",
+          "tooltip": "Specify the status of the task you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Group",
+          "name": "group",
+          "type": "text",
+          "description": "Specify the group of the task you want to create in TheHive Platform",
+          "tooltip": "Specify the group of the task you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Assignee",
+          "name": "assignee",
+          "type": "text",
+          "description": "Specify the assignee of the task you want to create in TheHive Platform",
+          "tooltip": "Specify the assignee of the task you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Additional Field",
+          "name": "additional_field",
+          "type": "json",
+          "description": "Specify the additional fields required for the use case. Provide the fields in JSON format.",
+          "tooltip": "Specify the additional fields required for the use case. Provide the fields in JSON format.",
+          "visible": true,
+          "editable": true,
+          "required": false
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "title": "Get Task",
+      "operation": "get_task",
+      "description": "Retrieves information about a specific task from The Hive platform based on the input parameters you have specified.",
```

Comment:
Please update description: Retrieves information about a specific task from The Hive platform based on the task ID you have specified.